Gordon360/ApiControllers/JobsController.cs

```diff
@@ -323,5 +323,198 @@ public IHttpActionResult DeleteClockIn()
             return Ok(result);
         }
 
+        //staff routes
+
+
+        /// <summary>
+        /// Get a user's active jobs
+        /// </summary>
+        /// <param name="details"></param>
+        /// <returns>The Staff's active jobs</returns>
+        [HttpPost]
+        [Route("getJobsStaff")]
+        public IHttpActionResult getJobsForStaff([FromBody] ActiveJobSelectionParametersModel details)
+        {
+            IEnumerable<ActiveJobViewModel> result = null;
+            int userID = GetCurrentUserID();
+            try
+            {
+                result = _jobsService.getActiveJobsStaff(details.SHIFT_START_DATETIME, details.SHIFT_END_DATETIME, userID);
+            }
+            catch (Exception e)
+            {
+                //
+                System.Diagnostics.Debug.WriteLine(e.Message);
+                return InternalServerError();
+            }
+            return Ok(result);
+        }
+
+        /// <summary>
+        /// Get a user's saved shifts
+        /// </summary>
+        /// <returns>The staff's saved shifts</returns>
+        [HttpGet]
+        [Route("getSavedShiftsStaff/")]
```

Comment:
yeah its so were expecting input i believe.

Reply:
Yeah I also don't really understand why it's that way either but there are part where student time sheets uses it and others they don't so it makes me think that it's their for a reason. I will remove the ones from the functions I added but I probably wont take from their functions since we know they work.